Code/Tools/SceneAPI/SceneCore/DataTypes/IManifestObject.h

```diff
@@ -22,11 +26,21 @@ namespace AZ
         }
         namespace DataTypes
         {
-            class IManifestObject
+            class SCENE_CORE_CLASS IManifestObject
             {
             public:
-                AZ_RTTI(IManifestObject, "{3B839407-1884-4FF4-ABEA-CA9D347E83F7}");
-                static void Reflect(AZ::ReflectContext* context);
+                SCENE_CORE_API virtual AZ::TypeId RTTI_GetType() const;
```

Comment:
I thought if SCENE_CORE_CLASS defines the declspec import/export, you don't need to do it for the members

Reply:
Well SCENE_CORE_CLASS is not defined on [Windows platforms](https://github.com/o3de/o3de/blob/a10351f38ddf50af2b01ed071c04cda52d6069ed/Code/Tools/SceneAPI/SceneCore/SceneCoreConfiguration.h#L18).
It is only set for non-Windows.